index.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-sequences */
+const titleInput = document.querySelector('#title-input');
+const authorInput = document.querySelector('#author-input');
+const addButton = document.querySelector('#add-button');
+const bookCollectionContainer = document.querySelector('#book-collection-container');
+const currentTime = document.querySelector('.current-time');
+const addNewLink = document.querySelector('.add-new-link');
+const formSection = document.querySelector('.form-section');
+const listLink = document.querySelector('.list-link');
+const list = document.querySelector('.list');
+const contactLink = document.querySelector('.contact-link');
+const contactInfo = document.querySelector('.contact-info');
+
+class BookLibrary {
+  constructor(title, author) {
+    this.title = title;
```

Comment:
Great though process!! 👍 
However, your constructor function was never the one responsible for the creation of the  titles and the authors of your books. It appeared that you were creating your books after some events(onSubmit) have been listened to. Rendering the constructor function useless.